x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/schema_callouts.tsx

```diff
@@ -129,3 +136,37 @@ export const ConfirmSchemaButton: React.FC = () => {
     </EuiButton>
   );
 };
+
+export const MissingSubfieldsCallout: React.FC = () => {
+  const { incompleteFields } = useValues(SchemaLogic);
+
+  return (
+    <EuiCallOut
+      iconType="alert"
+      color="warning"
+      title={`${incompleteFields.length} ${i18n.translate(
+        'xpack.enterpriseSearch.appSearch.engine.schema.incompleteFields.title',
+        { defaultMessage: 'field(s) are missing subfields' }
+      )}`}
+    >
+      <p>
+        {i18n.translate(
+          'xpack.enterpriseSearch.appSearch.engine.schema.incompleteFields.description',
+          {
+            defaultMessage:
+              'The field(s) are missing one or more subfields used by App Search. Some search features may not work until those subfields are added.',
+          }
+        )}{' '}
+        <a
+          href="https://www.elastic.co/guide/en/app-search/current/elasticsearch-engines-text-subfields-support-conventions.html"
+          data-test-subj="missingSubfieldsLearnMoreLink"
+          target="_blank"
+        >
+          {i18n.translate('xpack.enterpriseSearch.appSearch.engine.schema.incompleteFields.link', {
+            defaultMessage: 'Learn more.',
```

Comment:
Super nitpicky, but do we want "Learn more" to end with a period here?

Reply:
I think so, yeah. There's some copy with a similar treatment under [Known when to use the ending period](https://elastic.github.io/eui/#/guidelines/writing/guidelines). However, I did notice that just above the writing guide says **not** to use `(s)` to pluralize things so I'll fix that.